### PR TITLE
Fix pb inputs validation

### PR DESCRIPTION
**Pull request by michal-dagan**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6344

## Description
- Updated the regex value in collect_all_inputs_in_use method to `inputs\.[-\w ?!():]+`
- Added to playbook_valid_inputs.yml 3 cases

### CODE REVIEW

File: demisto_sdk/commands/validate/README.md [hunk 1]
```diff
@@ -40,8 +40,6 @@ Whether the validation should run only on the current branch's committed changed
 Whether the validation should run only on the current branch's staged files. This applies only when the **-g** flag is supplied.
 * **-iu, --include-untracked**
 Whether to include untracked files in the validation. This applies only when the **-g** flag is supplied.
-* **-i, --input**
-Path of file to validate specifically.
```
Comment (dorschw) [inline demisto_sdk/commands/validate/README.md:44]: intended?

File: demisto_sdk/commands/validate/README.md [hunk 2]
```diff
@@ -83,10 +81,6 @@ This will validate only changed files from content origin/master branch and will
 compatibility checks.
 <br><br>
 
-`demisto-sdk validate -j`
-This will validate all content repo files and including conf.json file.
-<br><br>
-
```
Comment (dorschw) [inline demisto_sdk/commands/validate/README.md:89]: intended?

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,6 +15,7 @@
 * Fixed an issue where **download** command failed when running on a beta integration.
 * Added the `DEMISTO_SDK_GRAPH_FORCE_CREATE` environment variable. Use it to force the SDK to recreate the graph, rather than update it.
 * Added support for code importing multi-level ApiModules to **lint**.
+* Fixed a bug where **validate** returned error on playbook inputs.
```
Comment (dorschw) [inline CHANGELOG.md:18]:
```suggestion
* Fixed an issue where **validate** returned error on playbook inputs.
```

what inputs? in which cases?

File: demisto_sdk/commands/validate/README.md (hunk 1 above)
Comment (michal-dagan) [reply demisto_sdk/commands/validate/README.md:44]: Yes, it already appears in lines 45-46

File: demisto_sdk/commands/validate/README.md (hunk 2 above)
Comment (michal-dagan) [reply demisto_sdk/commands/validate/README.md:89]: yes - 
when using -
```
demisto-sdk validate -j
You are using demisto-sdk 1.13.0.
Error: Option '-j' requires an argument.
```

**Review by bziser — APPROVED** [review]
Looks good
@YuvHayun  @dorschw Take a look

**Review by dorschw — APPROVED** [review]

**Comment by dorschw** [comment]
@eyalpalo, you [edited this code](https://github.com/demisto/demisto-sdk/pull/2256) last, please have a look

**Review by YuvHayun — APPROVED** [review]

**Review by adi88d — APPROVED** [review]
nice

**Pull request merged**